zb-classifier.js

```diff
@@ -1738,9 +1762,9 @@ class ZigbeeClassifier {
       console.log('      haElectricalEndpoint =', haElectricalEndpoint);
       console.log('    genBinaryInputEndpoint =', genBinaryInputEndpoint);
       console.log('      genLevelCtrlEndpoint =', genLevelCtrlEndpoint);
-      console.log('genLevelCtrlOutputEndpoint =', genLevelCtrlOutputEndpoint);
-      console.log('          genOnOffEndpoint =', genOnOffEndpoint);
-      console.log('    genOnOffOutputEndpoint =', genOnOffOutputEndpoint);
+      console.log('genLevelCtrlOutputEndpoint =', genLevelCtrlOutputEndpoints);
```

Comment:
nit: you pluralized the label for one of these, but not the other two

Reply:
done